x/tokenomics/keeper/settle_session_accounting_test.go

```diff
@@ -57,24 +63,290 @@ func TestSettleSessionAccounting_HandleAppGoingIntoDebt(t *testing.T) {
 
 	err := keepers.SettleSessionAccounting(ctx, &claim)
 	require.NoError(t, err)
-	// TODO_TEST: Need to make sure the application is unstaked at this point in time.
 }
 
 func TestSettleSessionAccounting_ValidAccounting(t *testing.T) {
-	t.Skip("TODO_BLOCKER(@Olshansk): Add E2E and integration tests so we validate the actual state changes of the bank & account keepers.")
-	// Assert that `suppliertypes.ModuleName` account module balance is *unchanged*
-	// Assert that `supplierAddress` account balance has *increased* by the appropriate amount
-	// Assert that `supplierAddress` staked balance is *unchanged*
-	// Assert that `apptypes.ModuleName` account module balance is *unchanged*
```

Comment:
@reviewer, this assertion seems to be incorrect to me. My understanding is that the account module balance MUST *decrease* as a result of the burn. I assumed this is copy-pasta and have the test making the modified assertion.